decidim-assemblies/app/controllers/decidim/assemblies/admin/members_controller.rb

```diff
@@ -13,7 +13,7 @@ def after_destroy_path
           members_path(current_assembly)
         end
 
-        def privatable_to
+        def participatory_space
```

Comment:
use `current_participatory_space` and remove this method.

Reply:
7ccd665e1a7c1df79774a91105b789166d1c38ea